Gemfile.lock

```diff
@@ -44,61 +44,60 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.3.1)
+    activeadmin (2.6.0)
```

Comment:
### [activeadmin](https://activeadmin.info)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.1 → 2.6.0

[_~~change-log~~, [source-code](http://github.com/activeadmin/activeadmin)_]





<details>
<summary>Commits</summary>

A change of **176** commits. See the full changes on [the compare page](https://github.com/activeadmin/activeadmin/compare/v2.3.1...v2.6.0).

These are the first 10 commits:
- (9736856) [Extract some common test logic for later reuse](https://github.com/activeadmin/activeadmin/commit/973685651276ae897011482353c3d2baa8cb20b8)
- (8e24320) [Make count query optimisation work when using decorate_with](https://github.com/activeadmin/activeadmin/commit/8e24320d8dff99d8ada9e17a7ca90be9e99708b1)
- (a2989f0) [Remove now unneeded workaround (#5859)](https://github.com/activeadmin/activeadmin/commit/a2989f0103ffd8212ec60f9bd3fc1f35d98bd9d9)
- (9a97cda) [Merge pull request #5811 from irmela/make-count-query-optimi…](https://github.com/activeadmin/activeadmin/commit/9a97cdab68ef803f817e555b97f20091ff747fef)
- (ece008d) [Completize missing Slovak translations (#5861)](https://github.com/activeadmin/activeadmin/commit/ece008dd015282503e3ec2b0f455f9a4bc056453)
- (83ddd79) [Make sure all PR links in changelog are implicit](https://github.com/activeadmin/activeadmin/commit/83ddd79b1451a1886f17c59bd83cf796b4783755)
- (4f3d4bf) [Sort contributors in changelog](https://github.com/activeadmin/activeadmin/commit/4f3d4bfb8f2df695696787b5b836d0795db3a9de)
- (dcff714) [Merge pull request #5863 from activeadmin/changeloging](https://github.com/activeadmin/activeadmin/commit/dcff71492f590d722b70d420a05dfdd9da1ec387)
- (5a7db30) [Avoid rendering huge associations (#5548)](https://github.com/activeadmin/activeadmin/commit/5a7db303904260f14aa653cc8280b91c56f92fa6)
- (efe50a4) [Bump tested 2.6 ruby patch level to 2.6.5](https://github.com/activeadmin/activeadmin/commit/efe50a408cb7047d52deed20de38d8f754ca924e)


</details>